docs/UserGuide.md

```diff
@@ -147,7 +147,34 @@ Format : `unmark i/STUDENT_ID DONE_TASK_INDEX`
 Example:
 * `unmark i/AXXXXXXXR 1`
     * Marks the first done task for the student with student ID AXXXXXXXR as undone.
-    
+
+### Editing a student's information: `edit`
+
+Edits a student's information in TAPA.
+
+Format: `edit STUDENT_INDEX [i/STUDENT_ID] [n/STUDENT_NAME] [m/MODULE_CODE] [p/PHONE_NUMBER] [h/TELEGRAM_HANDLE] [e/EMAIL_ADDRESS]​`
+
+* The index of the student to be edited is a compulsory field.
+* The student’s matriculation number, name, module code, phone number, telegram handle, and email address fields are optional and can be excluded.
+* An error message will be displayed to the user if the specified index is a negative number or larger than the number of students in TAPA.
+
+Example:
+* `edit 10 m/CS2103T p/98765432 t/@john e/E0123456Z`
```

Comment:
```suggestion
* `edit 10 m/CS2103T p/98765432 t/johnnn e/e0123456@u.nus.edu`
```
I realised there were typos in our original draft user guide for the format of edit and add... (will change the one for add later)

Reply:
ohh, I see, I wasn't too sure which email format to use in the user guide but I'll commit your suggestions first! 👍